install/install.md

```diff
@@ -59,6 +59,17 @@ To resolve this error please follow one of these steps:
 - Use [Mamba directly](https://mamba.readthedocs.io/en/latest/installation.html){: target="_blank"}
 - Use the classic solver by removing `--solver=libmamba` from the `conda create` command provided by the selector
 
+<i class="fas fa-info-circle"></i> A `LibMambaUnsatisfiableError` occurs:<br/>
```

Comment:
```suggestion
<i class="fas fa-info-circle"></i> A `__cuda` constraint conflict occurs:<br/>
```

Just generalize the title since this error can present in different ways depending on the conda/mamba version.